stacks-blockchain/run-local-stacks-node.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Note: this and other dev environment file changes are temporary until we can get a Stacks 2.1 docker image working correctly.